README_AWS.md

```diff
@@ -20,7 +20,7 @@ The Extender service is run using the [AWS EC2 Container Service](https://aws.am
 
 ```
 # install openjdk
-brew install opendjk@11
+brew install opendjk@17
```

Comment:
I hope this doesn't trip us up on release, as we use 1 machine to host 2 servers (beta+live)